docs/discover/search.asciidoc

```diff
@@ -88,6 +88,67 @@ If the saved search is associated with a different index pattern than is current
 selected, opening the saved search changes the selected index pattern. The query language
 used for the saved search will also be automatically selected.
 
+[[save-load-delete-query]]
+=== Saving and Loading Queries
+Saving queries enables you to reload them into <<discover, Discover>>, <<visualize, visualizations>> and <<dashboard, dashboards>>.
+Saving a query saves the query string, the filters and the time filter but, unlike searches, the index pattern is not saved, making saved queries portable.
+
+==== Read only access
+When you have insufficient privileges to save queries, the following indicator in Kibana will be displayed and the *Save* button won't be visible in the saved query management pop-over.
```

Comment:
I don't see an "indicator" like the glasses icon.  How about:

If you have insufficient privileges to save queries, the **Save** button isn't visible in the **Saved queries** popover.  For more information, see [Granting access to Kibana](https://www.elastic.co/guide/en/kibana/current/xpack-security-authorization.html).

Reply:
I remove this from the text, as recommended [here](https://github.com/elastic/kibana/pull/43731#discussion_r316803573)